drivers/mysql.go

```diff
@@ -442,3 +449,18 @@ func (db *MySQL) SetProvider(provider string) {
 func (db *MySQL) GetProvider() string {
 	return db.Provider
 }
+
+func (db *MySQL) formatTableName(tableName string) string {
+	splittedTableName := strings.Split(tableName, ".")
+
+	if len(splittedTableName) == 1 {
+		return tableName
```

Comment:
You could return this


```suggestion
		return fmt.Sprintf("`%s`", table)
```